contracts/evoting/evoting.go

```diff
@@ -821,6 +834,79 @@ func (e evotingCommand) manageAdminForm(snap store.Snapshot, step execution.Step
 	return nil
 }
 
+// manageVotersForm implements commands.
+// It performs the ADD or REMOVE VOTERS/OWNERS command
+func (e evotingCommand) manageOwnersVotersForm(snap store.Snapshot, step execution.Step) error {
+	msg, err := e.getTransaction(step.Current)
+	if err != nil {
+		return xerrors.Errorf(errGetTransaction, err)
+	}
+
+	var form types.Form
+	var formID []byte
+
+	txAddVoter, okAddVoter := msg.(types.AddVoter)
+	txRemoveVoter, okRemoveVoter := msg.(types.RemoveVoter)
+	txAddOwner, okAddOwner := msg.(types.AddOwner)
+	txRemoveOwner, okRemoveOwner := msg.(types.RemoveOwner)
```

Comment:
I hoped to reduce this to
```
tx, ok, func, msg := determineTypeSomehow()
form, formID, err = e.getForm(tx.FormID, snap)
if err != nil {
			return xerrors.Errorf(errGetForm, err)
}
err = func(tx.UserID)
if err != nil {
return xerrors.Errorf(msg, err)
}
```
but I don't really see a nice way to do it in Golang :/ so we can merge it like this for the time being